manual/als_help.html

```diff
@@ -0,0 +1,17 @@
+<HTML>
+<HEAD>
+<TITLE>ALS Prolog Help</TITLE>
+<link rel="stylesheet" type="text/css" href="alshelp.css">
+<HEAD>
+<style>
+#menu li {
+    list-style-type:none;
+}
+</style>
+</HEAD>
+<BODY CLASS="body">
+  <IFRAME SRC="package_nav.html"  id="left_top" CLASS="left_top" title="alspackages" scrolling="no"></IFRAME>
```

Comment:
Matching IDs and CLASSes is confusing. Only the IDs are referenced by styles or used by TOC, so remove CLASSes.

The SCROLLING attribute is deprecated, but can be replaced with CSS.

You might consider using semantic ID names rather then positional names (e.g. packages, toc, content).